scss/__tests__/generics.test.js

```diff
@@ -3,7 +3,11 @@ const path = require('path');
 
 test('default font-path', () => {
   // Compile against top-level entrypoint as we're testing settings
-  const output = sass.compile('scss/lib.scss').css.toString();
+  const output = sass
+    .compile('scss/lib.scss', {
+      logger: sass.Logger.silent,
+    })
```

Comment:
The `logger` change is just to better manage the test output for these sass checks